script/yaki

```diff
@@ -57,6 +57,14 @@ Commands:
     - If JOIN_TOKEN_CACERT_HASH is not provided, it will join using --discovery-token-unsafe-skip-ca-verification.
     - Example: JOIN_URL=<control-plane-endpoint>:<port> JOIN_TOKEN=<token> JOIN_TOKEN_CACERT_HASH=sha256:<hash> KUBERNETES_VERSION=v1.30.5 yaki join
 
+  upgrade: Upgrade Kubernetes components to specified version
+    - This command upgrades kubeadm, kubelet, and kubectl to the specified version.
+    - Requires: KUBERNETES_VERSION, APPLY (optional)
+    - If APPLY=true, runs 'kubeadm upgrade apply' (for control plane nodes)
+    - If APPLY is not set or false, runs 'kubeadm upgrade node' (for any node)
```

Comment:
This is a bit misleading unless user has a previous knowledge of `kubeadm`: is it possible understanding from the script if the node is a Control Plane or a Worker Node?